Tools/MediaDriverTools/UMDPerfProfiler/Readme.txt

```diff
@@ -0,0 +1,19 @@
+Introduction
+    The purpose of UMD Perf Profiler is to collect the performance data of media tasks, which can help us to know the workloads on GPU side. You can get the media tasks' timing, FPS and concurrency between GPU engines. 
+
+Usage
+• Step1: Build and install the latest internal-release version driver
+
+• Step2: Update registry file
+    Copy the igfx_user_feature.txt into /etc/ directory.
+    You will see some keys with default values in igfx_user_feature.txt.
+    -	Perf Profiler Buffer Size – Size of Perf profiler buffer, if not set will use the default value.
+    -	Perf Profiler Enable  - Enable/Disable UMD Perf Profiler, 1 – Enable, 0 – Disable
+    -	Perf Profiler Output File Name – The name of Perf Profiler output, if not set will use the default value.
+    
+• Step3: Run your test case
+    When finish your test case, you will see a bin file under your working directory which is named as set by “Perf Profiler Output File Name” in registry. If you didn’t set this key, the default name should be “linux_perf_out.bin”
+
+• Step4: Parser you bin file using MediaPerfParser
+    You will get the performance report by running ./MediaPerfParser linux_perf_out.bin.
```

Comment:
How to find MediaPerfParser tools?

Reply:
There's a binary file in this change, but isn't listed here, I think you will see it in the same folder once the request is approved.